tests/Unit/MerchantCenter/MerchantCenterServiceTest.php

```diff
@@ -524,13 +524,7 @@ public function test_get_setup_status_step_paid_ads() {
 					'countries' => [ 'GB' ],
 				],
 				[],
-				[
-					'website_live'            => true,
-					'checkout_process_secure' => true,
-					'payment_methods_visible' => true,
-					'refund_tos_visible'      => true,
-					'contact_info_visible'    => true,
-				]
+				[]
```

Comment:
One thing I wanted to mention here was that this test data is not a valid replacement. Because we no longer use the function `checked_pre_launch_checklist` the options are not fetched a fourth time.

We can confirm that by checking specifically for how many times it's called by using something like:
```php
$this->options->expects( $this->exactly( 4 ) )->method( 'get' )
```
Which will fail with a message:
> Expectation failed for method name is "get" when invoked 4 time(s).
> Method was expected to be called 4 times, actually called 3 times.

So I'd suggest to actually check that it's invoked 3 times as well as remove the parameters and response for the fourth time.

```php
		$this->options->expects( $this->exactly( 3 ) )->method( 'get' )
			->withConsecutive(
				[ OptionsInterface::MC_SETUP_COMPLETED_AT, false ],
				[ OptionsInterface::TARGET_AUDIENCE ],
				[ OptionsInterface::MERCHANT_CENTER, [] ]
			)->willReturnOnConsecutiveCalls(
				false,
				[
					'location'  => 'selected',
					'countries' => [ 'GB' ],
				],
				[]
			);
```

Reply:
👍🏻 @kt-12 can you make these updates and request another review from me?